src/tracks/track.entity.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+
+@Entity('tracks')
+export class Track {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+  
+  @Column({ type: 'varchar', length: '60' })
```

Comment:
That doesn't fit 🤫

<img width="1402" alt="screenshot_2020-05-13_10-38-29@2x" src="https://user-images.githubusercontent.com/11699655/81790631-f2da7680-9505-11ea-8f02-da0dd1dc566e.png">